packages/protocol/contracts/AirnodeParameterStore.sol

```diff
@@ -77,8 +77,8 @@ contract AirnodeParameterStore is RequesterStore, IAirnodeParameterStore {
     {
         bytes32 withdrawalRequestId = keccak256(abi.encodePacked(
             requesterIndexToNoWithdrawalRequests[requesterIndex]++,
-            requesterIndex,
-            address(this)
```

Comment:
I was using the contract address in the hash just to scramble the ID a bit more. I figured it's no longer needed with the chain ID (and it wasn't needed in the first place anyway).